package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "pgcc",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "format": "prettier --write ."
+  },
+  "dependencies": {
+    "@chakra-ui/react": "^1.8.6",
+    "@emotion/react": "^11.8.2",
+    "@emotion/styled": "^11.8.1",
+    "axios": "^0.26.1",
+    "framer-motion": "^6.2.8",
+    "next": "12.1.0",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-hook-form": "^7.28.1",
+    "react-icons": "^4.3.1",
+    "react-query": "^3.34.19",
+    "react-toastify": "^8.2.0"
+  },
+  "devDependencies": {
```

@caiotheodoro acabou tirando as dependências de `dependencies` mas faltou incluir em `devDependencies`

```suggestion
  "devDependencies": {
  "commitizen": "^4.2.4",
    "framer-motion": "^6.2.8",
    "git-cz": "^4.8.0",
```